packages/semver/src/builders/version/builder.spec.ts

```diff
@@ -141,6 +155,84 @@ describe('@jscutlery/semver:version', () => {
     expect(output).toEqual(expect.objectContaining({ success: false }));
   });
 
+  describe('Plugins', () => {
```

Comment:
@edbzn maybe we should move this describe in a distinct file.
There is already too much mocking in the parent describe `@jscutlery/semver:version` 😅.
This could be a dedicated suite where we would only mock `tryBump` and an internal `version` function that calls `versionProject` or `versionWorkspace`.

Reply:
I moved the tests in a dedicated plugin.spec.ts file, it tests both the handler and the adapter at the same with an integration test, maybe this could be improved.